recipes/r-ngsplot/build.sh

```diff
@@ -2,10 +2,17 @@
 outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $outdir
 mkdir -p $PREFIX/bin
-cp -R * $outdir 
+cp -R {bin,database,example,galaxy,lib,LICENSE} ${outdir}/
 #Set up links for 
-for f in $outdir/*; do
-    ln -s $outdir/* $PREFIX/bin
+for f in ${outdir}/bin/*; do
+    ln -s ${f} ${PREFIX}/bin
     fbname=$(basename "$f")
-    chmod 0755 ${PREFIX}/bin/$fbname
-done
\ No newline at end of file
+done
+
+#Activate/Deactivate dir
+ACTIVATE_DIR=$PREFIX/etc/conda/activate.d
+DEACTIVATE_DIR=$PREFIX/etc/conda/deactivate.d
+mkdir -p $ACTIVATE_DIR
+mkdir -p $DEACTIVATE_DIR
+cp $RECIPE_DIR/scripts/activate.sh $ACTIVATE_DIR/ngsplot-activate.sh
+cp $RECIPE_DIR/scripts/deactivate.sh $DEACTIVATE_DIR/ngsplot-deactivate.sh
```

Comment:
To avoid conflicts, you should rename those to the exact name of the package, i.e., `r-ngsplot-...`. You can use the `${PKG_NAME}` variable for this, see, e.g.: https://github.com/conda-forge/toolchain-feedstock/blob/bf0bfa1f06f6592eb3e6ef8989339a005cfd93ef/recipe/install-toolchain.sh#L8

Reply:
Done :-)